service/model_manager/model_list.py

```diff
@@ -3,45 +3,66 @@
 import server
 from aiohttp import web
 import hashlib
+import shelve
+import threading
+import os
 
-comfy_path = os.path.dirname(folder_paths.__file__)
+# The path to the file where file_hash_dict will be saved
+FILE_HASH_DICT_PATH = os.path.join(os.path.dirname(__file__),"../../hash/file_hash_dict")
```

Comment:
i'm thinking if we should save this info on disk or in indexdb, 
I'm thinking in indexdb we have a table:
```
type Model {
civitAiModelID:string
filehash:string
filename
folderName
}
```

Reply:
comfy_path = os.path.dirname(folder_paths.__file__)
custom_nodes_path = os.path.join(comfy_path, 'custom_nodes')

lets use folder_paths to get the right comfy path

Reply:
indexdb is in browser, while the hash is calculated in python, it would add extra overhead and a lot of work to save it in indexdb

Reply:
since we are the one generated the hash, and we are the only one to use the hash, would it be more reasonable to save it inside `comfyui-workspace-manager ` folder

Reply:
yeah makes sense. i think we should def save one in disk, later in future, we'll see if we need to save a copy in indexdb